lib/B/Deparse.pm

```diff
@@ -4072,11 +4072,15 @@ sub pp_leavetrycatch {
     $catch->name eq "catch" or die "Expected catch as third child of leavetrycatch";
 
     my $catchblock = $catch->first->sibling;
-    $catchblock->name eq "scope" or die "Expected scope as second child of catch";
+    my $name = $catchblock->name;
+    unless ($name eq "scope" || $name eq "leave") {
+      "Expected scope or leave as second child of catch, got $name instead\n";
```

Comment:
Did you mean to maybe "return" or "die" this string?

Reply:
Bah, fixed